setup.cfg

```diff
@@ -15,6 +15,25 @@ project_urls =
     Source = https://github.com/spacetelescope/wfc3tools/
     Help = https://hsthelp.stsci.edu
 
+[options]
+python_requires = >=3.8
+setup_requires =
+    setuptools_scm
+install_requires =
+    astropy>=4.2.0
+    numpy>=1.21.0
+    matplotlib>=3.5.1
+    stsci.tools>=4.0.1
+    scipy>=1.8.0
+
+[options.extras_require]
+docs =
+    sphinx
+    sphinx-automodapi
+    sphinx-rtd-theme
+    stsci-rtd-theme
+    sphinx-astropy
+
 [build-sphinx]
```

Comment:
I don't think you need `build-sphinx` either. Do you use it?

Reply:
i'm not sure, i'll take it out for now and i'm going to be making another PR with some changes for documentation